administrator/components/com_joomlaupdate/tmpl/update/download.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_joomlaupdate
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
```suggestion
 * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
```
new file so set the correct date